oz/core/actions.py

```diff
@@ -107,7 +119,15 @@ def server():
             signal.signal(signal.SIGTERM, functools.partial(_shutdown_handler, http_srv))
             signal.signal(signal.SIGINT, functools.partial(_shutdown_handler, http_srv))
 
-        tornado.ioloop.IOLoop.instance().start()
+        if server_type == "asyncio":
```

Comment:
It looks like the implementation varies quite a bit from one server type to another. Rather than block this further in investigating how to do this properly for asyncio/twisted, I've just enabled it for tornado ioloops.